files/requirements.py

```diff
@@ -107,6 +113,11 @@ def setup_python(python: Python, source_directory: str, final: bool) -> None:
     display.section(f'Checking pip integrity for Python {python.version}')
     pip.check()
 
+    if python.version == '3.12':
+        display.info(f'Skipping checking PyYAML for libyaml support for Python {python.version}')
+        display.info(f'Skipping checking coverage C extension support for Python {python.version}')
```

Comment:
Instead of skipping the `coverage` check here, we should be able to add Python 3.12 to ansible-test:

https://github.com/ansible/ansible/blob/cde15f3c8158467a96023a9cffcba4bc0a207b0f/test/lib/ansible_test/_internal/coverage_util.py#L72

Reply:
I need to review https://github.com/ansible/ansible/pull/81077 first. If we can get it merged this week, then that's probably better than bypassing the check here.

Reply:
Lets go ahead and try with the existing `coverage` version, even though it's missing wheels for Python 3.12. We should be able to do that by expanding the version range in ansible-test:

https://github.com/ansible/ansible/blob/cde15f3c8158467a96023a9cffcba4bc0a207b0f/test/lib/ansible_test/_internal/coverage_util.py#L72

That way `coverage` will be installed in the container, instead of at test time. It just means the container build will have to build the wheel first, since it won't be able to download one.

That change will cause a minor conflict with https://github.com/ansible/ansible/pull/81077 -- but it's not ready to merge at the moment, due to what appears to be an issue with the new `coverage` version not behaving as documented.